python/CMakeLists.txt

```diff
@@ -80,6 +80,8 @@ if(TORCH_MLIR_ENABLE_JIT_IR_IMPORTER)
       LIBTORCH_CACHE=${LIBTORCH_CACHE}
       LIBTORCH_VARIANT=${LIBTORCH_VARIANT}
       CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}
+      CMAKE_C_COMPILER_LAUNCHER=${CMAKE_C_COMPILER_LAUNCHER}
+      CMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_CXX_COMPILER_LAUNCHER}
```

Comment:
I had some trouble testing passing the compiler flags from outside the build_libtorch.sh build and that is the reason I have a PT_C_COMPILER / PT_CXX_COMPILER. If this works that would be great to get rid of that in follow on commits.

Reply:
FWIW We have to move all of these out with SET(ENV{LIBTORCH_VARIANT} ${LIBTORCH_VARIANT}) outside of the execute_process() so the builder would pass. I think the env vars set with -E env get lost when running inside a pip wheel sandbox.